src/components/userProfile/UserProfile.js

```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
       );
     }
 
+    let { scale } = this.state;
+    let imageStyle = { transform: [{ scale }] };
+
     return (
       <View style={{
         justifyContent: 'space-between',
         flexDirection: 'column',
         flex: 1,
       }}>
-        <ScrollView>
+        <ScrollView
+            scrollEventThrottle={16}
+            onScroll={this.handleScroll.bind(this)}>
           <StatusBar
             backgroundColor = "transparent"
             barStyle = "light-content"
             networkActivityIndicatorVisible={false}
           />
-          <LinearGradient style={styles.profileImgGradient} start={[0.0, 0.25]} end={[0.5, 1.0]}
-            colors={['#546979', '#08233a']}>
-            <Image style={styles.profileImage}
-              source={this.state.profileImage} />
-          </LinearGradient>
-          <Image style={styles.bookmarkIcon}
-            source={require('../../resources/icon-bookmark.png')}/>
+          <Animated.View
+              {...this.panResponder.panHandlers}
+              style={[this.state.pan.getLayout(), imageStyle]}>
+            <LinearGradient
+                style={styles.profileImgGradient} start={[0.0, 0.25]} end={[0.5, 1.0]}
+                colors={['#546979', '#08233a']}>
+              <Image
+                  style={styles.profileImage}
+                  source={this.state.profileImage} />
+            </LinearGradient>
+          </Animated.View>
+          <Image
+              style={styles.bookmarkIcon}
+              source={require('../../resources/icon-bookmark.png')}/>
```

Comment:
1. A space is required before closing bracket

2. The closing bracket must be aligned with the line containing the opening tag (expected column 11 on the next line)